flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/JsonFormatFactory.java

```diff
@@ -151,4 +151,15 @@ public String factoryIdentifier() {
         options.add(ENCODE_DECIMAL_AS_PLAIN_NUMBER);
         return options;
     }
+
+    @Override
+    public Set<ConfigOption<?>> mergeableOptions() {
```

Comment:
Call this `forwardOptions` similar to helper? Because those are not only mergeable but will be forwarded.